### ISSUE TITLE
Surplus implementation doesn't have aria attributes

### ISSUE THREAD

**Issue by localvoid** [krausest/js-framework-benchmark#213]
@adamhaile 

React and probably all other implementations:
 - [Main](https://github.com/krausest/js-framework-benchmark/blob/8b9f6613cf3b7e94f7bcef1a784a2db49211b368/react-v15.5.4-keyed/src/Main.jsx#L128)
 - [Row](https://github.com/krausest/js-framework-benchmark/blob/8b9f6613cf3b7e94f7bcef1a784a2db49211b368/react-v15.5.4-keyed/src/Row.jsx#L39)

Surplus:
 - [Main](https://github.com/krausest/js-framework-benchmark/blob/8b9f6613cf3b7e94f7bcef1a784a2db49211b368/surplus-v0.4.0-keyed/src/view.tsx#L43)
 - [Row](https://github.com/krausest/js-framework-benchmark/blob/8b9f6613cf3b7e94f7bcef1a784a2db49211b368/surplus-v0.4.0-keyed/src/view.tsx#L57)

**Comment by leeoniya** [comment]
yep, it's required [1]. ping @adamhaile.

[1] https://github.com/krausest/js-framework-benchmark/issues/9#issuecomment-232443783

**Comment by adamhaile** [comment]
Oops, yep, that's a goof.  I don't remember removing the attribute, but I can believe it.  The underlying issue is that surplus' currently-released compiler only supports a subset of JSX, specifically the part that overlaps with surplus' earlier template syntax.  Attributes that aren't available as properties, like aria-*, aren't covered.  I did the initial implementation of the benchmark one evening months ago.  It was actually the first thing I wrote using JSX.  I'll bet I ditched aria-hidden then.  I should have fixed that before submitting but honestly didn't remember.  Thanks for catching it @localvoid.

I'm currently rewriting the surplus compiler to support more of JSX (spreads, the style object, attributes, etc).  I was planning to resubmit when that was done.  I can roll the aria-hidden fix into that or push a patch, whatever @krausest  prefers.
